### ISSUE TITLE
[runtimes] API to add all attachments of a skin to another (empty/new) skin

### ISSUE THREAD

**Issue by badlogic** [EsotericSoftware/spine-runtimes#990]
See http://esotericsoftware.com/forum/Generating-a-skin-at-runtime-libGDX-9092?p=41209#p41209

```
/** Adds all attachments from the specified skin to this skin. */
public void addAttachments (Skin skin) {
   for (Entry<Key, Attachment> entry : skin.attachments.entries())
      addAttachment(entry.key.slotIndex, entry.key.name, entry.value);
}
```

See also http://esotericsoftware.com/forum/Combinig-skins-in-C-runtime-9274

- [ ] spine-libgdx

---
- [ ] spine-csharp
- [ ] spine-xna
- [ ] spine-monogame
- [ ] spine-unity
- [ ] spine-tk2d

---
- [ ] spine-c
- [ ] spine-sfml
- [ ] spine-cocos2d-objc
- [ ] spine-cocos2dx
- [ ] spine-ue4

---
- [ ] spine-ts core
- [ ] spine-ts webgl
- [ ] spine-ts canvas
- [ ] spine-ts threejs

---
- [ ] spine-as3
- [ ] spine-starling

---
- [ ] spine-lua
- [ ] spine-love
- [ ] spine-corona

**Comment by badlogic** [comment]
See also https://github.com/EsotericSoftware/spine-runtimes/issues/841. Not sure if we should tackle this shallow copy use case or the full monty.

**Comment by badlogic** [comment]
Closed in favor of #841
